### PR TITLE
Gamemode Vote in the Lobby

### PR DESCRIPTION
**Pull request by TalkingCactus**
Calls a gamemode vote in the lobby. Currently includes secret and extended. If nobody votes or there is a tie, the vote is inconclusive and the gamemode does not change. Adds a new virus to the shuttle loan event.

image

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks great, we needed this.

File: code/modules/events/shuttle_loan.dm [hunk 1]
```diff
@@ -140,7 +140,7 @@
 				new /obj/effect/spider/stickyweb(T)
 
 			if(ANTIDOTE_NEEDED)
-				var/virus_type = pick(/datum/disease/beesease, /datum/disease/brainrot, /datum/disease/fluspanish)
+				var/virus_type = pick(/datum/disease/beesease, /datum/disease/brainrot, /datum/disease/fluspanish, /datum/disease/transformation/xeno)
```
Comment (ghost) [inline code/modules/events/shuttle_loan.dm:143]: WAIT WHAT ARE YOU DOING

Comment (Boy-Named-Sue) [reply code/modules/events/shuttle_loan.dm:143]: Clever girl.

Comment (TalkingCactus) [reply code/modules/events/shuttle_loan.dm:143]: This actually doesn't even work, so rip.

**Pull request merged**